README.md

```diff
@@ -12,6 +12,20 @@ You can also [install and run the notebooks on a local machine](#local-setup).
 This is helpful if you can't access mybinder.org, or if you want to
 [contribute to this tutorial](#contributing-to-this-tutorial).
 
+## Setup with Nebari for Scipy 2023
```

Comment:
We can remove this section after SciPy. :)

Reply:
It would be great if you could help me remember! 😅